string data_path
uint16 data_point
uint16 frame_count
robot_msgs/RobotConfiguration conf
geometry_msgs/PoseStamped camera_target
geometry_msgs/PoseStamped world_target
---
uint16 captured_frames
